source/auth/auth.md

```diff
@@ -375,7 +375,7 @@ When connected to MongoDB 3.2 or earlier:
 
 - source
 
-    MUST be "$external". Defaults to `$external`.
+    MUST be "$external". Defaults to `$external\`.
```

Comment:
Thanks for fixing these

Reply:
I don't think this is right either - it's failing lint for me locally